### ISSUE TITLE
Build error with updated packages

### ISSUE THREAD

**Issue by Surfrat** [rid00z/FreshMvvm#285]
Hi,

I updated all nuget packages and have the following problems:

1. When I build I have the following UWP error:

![image](https://user-images.githubusercontent.com/2066487/63934220-1edac800-ca5b-11e9-9aff-73f3c6956480.png)

2. If I unload the UWP project I can build but Android fails with this:

![image](https://user-images.githubusercontent.com/2066487/63934381-8264f580-ca5b-11e9-906d-9d081fed39ee.png)

iOS runs fine.

Windows: 1903 (18362.295)
VS 2019 v16.2.3

I have searched for solutions to the above problems with no luck. Attached is the updated project.

Can you help in getting this to run?

Thanks.
[FreshMvvm-master-updated.zip](https://github.com/rid00z/FreshMvvm/files/3555199/FreshMvvm-master-updated.zip)

**Comment by libin85** [comment]
the error that you have seems to be with the solution properties. make sure the android project is deployable.

**Comment by Surfrat** [comment]
Thanks for the reply. It is.... Please load up my test project to see what the issues are. I have already spent a couple of hours on this.

**Comment by libin85** [comment]
@Surfrat : i will give it a try tonight if i get some time

**Comment by romeop555** [comment]
i have exactrly the same probem in the first post....

**Comment by libin85** [comment]
@Surfrat @romeop555 : I have downloaded your project tried both android and ios and it works all good for me. UWP, I have not tested yet, will try that on my windows machine later. The only difficulty that I had was restoring the packages in visual studio.  Once I restarted the Visual studio couple of times, that was fixed too.

**Comment by Surfrat** [comment]
Thank you for testing. I assume it was on a Mac. Please try the Android project on the Windows side as well.

**Comment by libin85** [comment]
Yes, it was on Mac. I will test it on windows later today if i get some time

**Comment by libin85** [comment]
@Surfrat @romeop555 : i have tested the project on windows. Android works all good. For UWP, I did get the same error as you. The error sounded like the UWP project was not able to get the reference of FreshMvvm from FreshMvvmApp project . So i first removed the FreshMvvmApp reference and added it back and it was still not working. I then added FreshMvvm reference directly to UWP project too. And it works now. Hope that helps

**Comment by Surfrat** [comment]
Thank you very much for looking at this. Could you please post your working Windows version of the project here so I can see the differences?

Also would you consider updating the source to reference updated libraries.?

**Comment by Surfrat** [comment]
I would appreciate it if you could post your working project. Thanks

**Comment by Surfrat** [comment]
It appears that @libin85  has left the building :)

The reason for my post is that as a new user, one of the first things you will do with this project, is update all the Nuget packages. I have been using FreshMVVM for about 2 years and my project is fully updated and has no issues.

I am a big fan of FreshMVVM and recently watched Michael Ridland on the Xamarin show. I am a little disappointed that FreshMVVM is not getting much attention here.

To help out any newcomers I have added a version that us fully updated and runs on all platforms. It would be nice if the source was updated as it still references Xamarin Forms 3.

Note that I removed the UI projects from the FreshMVVM project.
Build the FreshMVVM project.
Load theFreshMVVMApp project and build that.
Hopefully all the UI projects will run.

Thanks

Mark.
[FreshMvvm-master-updated-Working.zip](https://github.com/rid00z/FreshMvvm/files/3620419/FreshMvvm-master-updated-Working.zip)

**Comment by rid00z** [comment]
@Surfrat Thanks for your concern. I agree that it's good to ensure that running FreshMvvm from the code base is a smooth experience. Some of these issue will be easy to resolve, I will take a look.
